db/seeds.d/07-provisioning_templates.rb

```diff
@@ -9,6 +9,7 @@
       if t.template != contents
```

Comment:
If you change it to `if t.template != contents || t.vendor != 'Foreman'` it updates existing templates too even if their content is the same. We're updating their content anyway so it's safe to consider these are shipped by Foreman.